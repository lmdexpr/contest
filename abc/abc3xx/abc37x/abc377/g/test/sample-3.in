8
at
atatat
attat
aatatatt
attattat
ttatta
tta
tt
