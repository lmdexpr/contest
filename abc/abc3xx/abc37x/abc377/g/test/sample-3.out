2
4
3
8
3
6
3
1
